spec/predicates/vuln.md

```diff
@@ -0,0 +1,173 @@
+# Predicate type: Vulnerabilities
+
+Type URI: https://in-toto.io/attestation/vulns/attribute-report
```

Comment:
Nit: Any reason the "attribute-report" part is in the TypeURI? Shorter/simpler would be preferable.